.lein-failures

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Try and take out this file from git!

Reply:
I put the file names in .gitignore but I used "git rm --cached <FILENAME>" this time. I think it does the trick.